cfme/containers/image.py

```diff
@@ -122,7 +126,33 @@ def compliant(self):
             raise ValueError("{} is not a known state for compliance".format(text))
 
 
+@attr.s
+class ImageCollection(BaseCollection):
+    """Collection object for :py:class:`Image`."""
+
+    ENTITY = Image
+
+    def all(self):
+
```

Comment:
remove this empty line or preferably write docblock

Reply:
Done